pkg/contracts/script/VerifyTest.s.sol

```diff
@@ -37,30 +37,8 @@ import {BaseMultiChain} from "./BaseMultiChain.s.sol";
 contract VerifyTest is BaseMultiChain {
     using stdJson for string;
 
-    function run(string memory network) public virtual override {
-        // get PK from env
-        councilMemberPKEnv = vm.envUint("PK");
-        if (councilMemberPKEnv == 0) {
-            revert("PK not set");
-        }
-
-        vm.startBroadcast(pool_admin());
-
-        if (bytes(network).length != 0) {
-            CURRENT_NETWORK = network;
-        }
-
-        string memory json = getNetworkJson();
-
-        uint256 chainId = json.readUint(getKeyNetwork(".chainId"));
-        string memory name = json.readString(getKeyNetwork(".name"));
-        SENDER = json.readAddress(getKeyNetwork(".ENVS.SENDER"));
-
-        console2.log("name: %s", name);
-        console2.log("sender: %s", SENDER);
-        console2.log("chainId : %s", chainId);
-
-        allo_proxy = json.readAddress(getKeyNetwork(".ENVS.ALLO_PROXY"));
+    function runCurrentNetwork(string memory networkJson) public virtual override {
+        allo_proxy = networkJson.readAddress(getKeyNetwork(".ENVS.ALLO_PROXY"));
```

Comment:
we have this just to check if allo exist?

Reply:
No its to pull the allo proxy address from the networks.json for the corresponding network, but was Felipe that did this to run contract verification